release/test-dependencies/actual.liferay-jenkins-ee.build.properties/build.properties

```diff
@@ -0,0 +1,2796 @@
+#
```

Comment:
@kiwm, can you please keep in the `build.properties` just what is strictly necessary for running the new tests created by you successfully? Please see this [commit](https://github.com/liferay/liferay-docker/commit/e30b22390d0c4e69dd53c06fa248ba5d25850b86) to get some inspiration. :wink: